CRM/Contribute/Form/Contribution/Confirm.php

```diff
@@ -2060,6 +2062,7 @@ public static function submit($params) {
     $paramsProcessedForForm = $form->_params = self::getFormParams($params['id'], $params);
```

Comment:
This function (submit()) is used primary by unit tests via the ContributionPage.submit api. AFAIK that api is only used by a barage of tests but there might be some live usages